doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -29,15 +29,15 @@ Execute in webcam:
 
 @code{.bash}
 
-$ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[PATH-TO-DARKNET]/yolo.weights --classes=object_detection_classes_pascal_voc.txt --width=416 --height=416 --scale=0.00392
+$ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[PATH-TO-DARKNET]/yolo.weights --classes=[PATH-TO-DARKNET]/data/coco.names --width=416 --height=416 --scale=0.00392 
```

Comment:
Please remove a trailing whitespace.

I think it's better to remove `[PATH-TO-DARKNET]` and replace them to simple `yolo.weights` and `yolo.cfg` and something like `/path/to/classes/names.txt`. As an example, you can mention that there is `coco.names` at Darknet's repository for models trained on COCO dataset. Otherwise it can mislead users that OpenCV uses Darknet.

Please add `--rgb` flag because all the YOLO models process RGB images.

Reply:
ok

Reply:
done